snaps_tests/demos_tests/test_ros.py

```diff
@@ -29,6 +30,12 @@ class ROSTestCase(snaps_tests.SnapsTestCase):
     @skipUnless(linux_distribution()[2] == 'xenial',
                 'This test fails on yakkety LP: #1614476')
     def test_ros(self):
+        if snapcraft.ProjectOptions().deb_arch == 'arm64':
+            # https://bugs.launchpad.net/snapcraft/+bug/1662915
+            self.expectFailure(
+                'There are no ros-indigo packages in the ros archive',
```

Comment:
To be explicit, there are no arm64 ROS Indigo packages in the ROS archive (not just no packages).